.github/workflows/nexus-publish-snapshots.yml

```
@@ -40,4 +46,3 @@ jobs:
         env:
           MAVEN_REPO_USERNAME: ${{ secrets.NEXUS_USERNAME }}
           MAVEN_REPO_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
```

```suggestion
          MAVEN_REPO_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
          
```
You can configure your IDE to always end files this way